coderd/database/migrations/000102_workspace_agent_stats_types.up.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE	workspace_agent_stats ADD COLUMN connection_median_latency_ms bigint DEFAULT 0 NOT NULL;
```

Comment:
Skulle this default value be -1 same as in the code?

Reply:
Yup, good catch!